inbound/converter/tests/test_edifact_to_fhir.py

```diff
@@ -0,0 +1,44 @@
+import unittest
```

Comment:
This test doesn't run in the build due to missing __init__.py files in parent directories

When I run it manually it fails

Reply:
the __init__.py is added